luigi/task.py

```diff
@@ -289,6 +289,9 @@ def __init__(self, *args, **kwargs):
         self.task_id = task_id_str(self.task_family, self.to_str_params(only_significant=True))
         self.__hash = hash(self.task_id)
 
+        self.status_message = None
+        self._status_message_callback = None
```

Comment:
Nevermind the first paragraph. User is not overriding `_status_message_callback`. :)